src/modifiers/distribute_along_even.gd

```diff
@@ -3,9 +3,12 @@ extends "base_modifier.gd"
 
 
 export(float) var interval = 1.0
-export(float) var offset = 0.0
+export(float, 0, 100) var offset = 0.0
 export(bool) var align_to_path = false
-
+export(int) var align_up_axis = 0
```

Comment:
This should probably default to 1 (Y axis) as it will be the most common scenario

Reply:
Yes, I initially did have it as the Y axis, but I was having issues with the proper axis not being returned given the axis enum